src/compose/ComposeBox.js

```diff
@@ -138,12 +139,41 @@ export default class ComposeBox extends PureComponent {
     this.setState({ message: input });
   };
 
+  componentWillUnmount() {
+    const { drafts } = this.props;
+    const thisNarrow = JSON.stringify(this.props.narrow);
```

Comment:
I'll do it here as it will simplify the code, but will they optimize the performance
As all we need to do is a key lookup in the state

Also considering the fact the draft will not be used in other components?

Reply:
Yep, just for simplifying the code. Perf difference will be negligible.